src/api/config/Constants.h

```diff
@@ -0,0 +1,17 @@
+
+
+#ifndef FIUBA_TALLER2_TALLERIFY_APP_SERVER_CONSTANTS_H
+#define FIUBA_TALLER2_TALLERIFY_APP_SERVER_CONSTANTS_H
+
+#define HTTP_GET "GET"
+#define HTTP_POST "POST"
+#define HTTP_PUT "PUT"
+#define HTTP_DELETE "DELETE"
+#define HTTP_OK 200
+#define HTTP_CREATED 201
+#define HTTP_BAD_REQUEST 400
+#define HTTP_NOT_FOUND 404
```

Comment:
Los defines prefijalos con el nombre de la clase tmb, acordate que no usas namespace con macros asi que podrias llegar a pisarlas sino (y arrancalas con _ tipo `_STATUS_CODE_OK` (asi es es c/c++ live with it)